pkg/dataplane/http/context.go

```diff
@@ -524,11 +524,18 @@ func (c *context) PutObject(putObjectInput *v3io.PutObjectInput,
 
 // PutObjectSync
 func (c *context) PutObjectSync(putObjectInput *v3io.PutObjectInput) error {
+
+	var headers map[string]string
+	if putObjectInput.Append {
```

Comment:
If the underlying header is "Range" - I imagine it has other uses other an appending.
Are we sure we want to limit the API to just a boolean of "append" or "don't append"?

Reply:
i am not sure you are right, here's what written in our documentation: https://igzdocsdev.wpengine.com/docs-dev/reference/latest-release/api-reference/web-apis/simple-object-web-api/data-object-operations/#appending-data